sdk/python/tests/vectors/all.py

```diff
@@ -215,10 +243,13 @@ def load_test_suites(blockchain):
 		VerifyTester(class_locator),
 		Bip32DerivationTester(class_locator),
 		Bip39DerivationTester(class_locator),
+		DeriveTester(class_locator),
```

Comment:
would move up to match source code order

Reply:
order seems to be bit _weird_, MosaicIdDerivationTester is before Bip tests, althought it is appended at the end of a list 😬